source/common/memory/heap_shrinker.cc

```diff
@@ -15,10 +15,10 @@ HeapShrinker::HeapShrinker(Event::Dispatcher& dispatcher, Server::OverloadManage
                            Stats::Scope& stats)
     : active_(false) {
   const auto action_name = Server::OverloadActionNames::get().ShrinkHeap;
-  if (overload_manager.registerForAction(action_name, dispatcher,
-                                         [this](Server::OverloadActionState state) {
-                                           active_ = (state == Server::OverloadActionState::Active);
-                                         })) {
+  if (overload_manager.registerForAction(
+          action_name, dispatcher, [this](Server::OverloadActionState state) {
+            active_ = (state == Server::OverloadActionState::saturated());
```

Comment:
variable naming consistency: active_ is true when state is saturated().  Should the variable be called saturated_?

Reply:
I was trying to touch this code as little as possible, hence the minimal change. I'd suggest that since the heap shrinker can't be partially active, it doesn't make sense to talk about saturation here, and to leave this as a boolean `active_`.